File: Go/01.Go-Basic-February-2022/09.advancedLoops-Lab/03.sumNumbers.go
package main

import "fmt"

func main() {
	var firstNumber int
	fmt.Scanln(&firstNumber)

	var sum int = 0

	for sum < firstNumber {
		var number int
		fmt.Scanln(&number)

		sum += number		
	}

	fmt.Print(sum)
}

File: JavaScript/01.JavaScript-Basic-February-2021/09.whileLoop-Lab/03.sumNumbers.js
function sumNumber(input) {
    let index = 1;

    let firstNum = Number(input[0]);
    let num = Number(input[index]);
    let sum = 0;

    while (sum < firstNum) {
        sum += num;
        index++;
        num = Number(input[index]);
    }
    console.log(sum);
}

sumNumber(["100", "10", "20", "30", "40"]);